spec/features/sort_message_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'Sort messages' do
+  scenario 'Shows all bookmarks sorted by date' do
+    Message.create(message: 'test_1')
+    Message.create(message: 'test_2')
+    Message.create(message: 'test_3')
+
+    visit('/')
+    click_button('Sort by date')
+    
+    expect(page).to have_content("test_1")
```

Comment:
This feature tests is a little tricky but, using capybara, you can grab all elements by, for example, class and that will return an array of elements in the same order as they appear on the page.  So then you can test for order.